src/main/views/template.njk

```diff
@@ -36,7 +36,7 @@
         },
         {
             text: navLinks.link2,
-            href: 'https://hmcts-sjp.herokuapp.com/sign-in.html'
```

Comment:
changing this affects user journey for the testing sessions, are BAs aware of this change?

Reply:
reverted